tests/asset.rs

```diff
@@ -521,6 +527,72 @@ async fn allow_transfer_assets_to_one_beneficiary() -> Result<()> {
                 .await?;
                 debug!("Accept response: {:#?}", &accept_details);
                 assert!(issued_assets.contains(&accept_details.id), "RGB IDs match");
+
+                info!("First transfer worked! Attempting second...");
+
+                assets_vault_details = get_assets_vault(
+                    &tmp_vault.rgb_assets_descriptor_xpub,
+                    &tmp_vault.rgb_udas_descriptor_xpub,
+                )
+                .await?;
+
+                debug!("New assets vault details: {assets_vault_details:?}");
+
+                info!("Get a second Blinded UTXO");
+                debug!("First was: {blinded_utxo1:?}");
+                let blinded_utxo2 = get_blinded_utxo(&assets_vault_details.assets_change_output);
+                assert!(blinded_utxo2.is_err(), "TODO: Expected error in getting a second blinded UTXO because no new change output has been made");
+                return Ok(());
+                debug!("Blinded UTXO 2: {blinded_utxo2:?}");
+
+                let transfers = vec![
+                    AssetTransfer {
+                        asset_contract: issued_asset.genesis.to_string(),
+                        asset_utxo: AssetUtxo {
+                            outpoint: assets_vault_details.assets_output.clone().unwrap(),
+                            terminal_derivation: "/0/0".to_string(), // TODO: Should we use /0/1 instead? If so, how? Or should we try to keep it simple for now and reuse addresses?
+                            commitment: "".to_string(),
+                        },
+                        asset_amount: SUPPLY,
+                        change_utxo: assets_vault_details.assets_change_output.clone().unwrap(),
```

Comment:
Here breaks because `asset_change_output` is None

Reply:
I'm not sure a check is needed for this in the test... For multiple transfers to work, new / replacement UTXOs need to be defined upon subsequent calls to this method.